utils/firestore.js

```diff
@@ -2298,8 +2298,32 @@ const getNotificationSpecByCategoryAndAttempt = async (category = "", attempt =
   return snapshot.empty ? null : snapshot.docs[0].data();
 };
 
+const validateKitAssemblyData = (data) => {
+    // Ensure that values are uppercase and that the appropriate values match
+    if(data[fieldMapping.returnKitId]) {
+        data[fieldMapping.returnKitId] = ('' + data[fieldMapping.returnKitId]).toUpperCase();
```

Comment:
What is the purpose of adding the empty strings in all these statements?

Reply:
I do it to make 100% sure it's a string before running toUpperCase, in case we end up with a number.